### ISSUE TITLE
Try convert: displays multiple instances , without being able to pick one

### ISSUE THREAD

**Issue by fasteddys** [dotnet/try-convert#426]
Hello Try-convert to .NET core, display all the install and hangs.

theres no way to select and instance, and specifying --tfm also hangs

**Comment by serp777** [comment]
if you're trying on VS 2022 don't bother, its completely broken. You might have to install vs 2019 to use it.

**Comment by AlexeyTurchaninov** [comment]
@serp777 it's not necessary, he just need to install [.NET 5 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/5.0) and try-convert will work.

**Comment by serp777** [comment]
> @serp777 it's not necessary, he just need to install [.NET 5 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/5.0) and try-convert will work.

Are you positive? I've tried .net core 3.1 SDK, .net 5, and .net 6 and there's always problem. Running on visual studio 2019 though made everything work great.

**Comment by AlexeyTurchaninov** [comment]
@Cpt-Falcon yes, it helped me. But to be honest, I had Visual Studio 2019 (without .NET 5 SDK), which I uninstalled before come to Visual Studio 2022.

**Comment by fxredeemer** [comment]
> @serp777 it's not necessary, he just need to install [.NET 5 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/5.0) and try-convert will work.

This worked fine for me without having VS 2019 installed

**Comment by fasteddys** [comment]
## Warning from Microsoft both visual studio & Microsoft updates, `NET 5 is no longer supported`

![image](https://user-images.githubusercontent.com/96149622/182466725-7124641c-48b0-44cf-bc5f-5432d05a5f35.png)
